lambda/requests-aggregated/requests-aggregated.js

```diff
@@ -1,30 +1,51 @@
+const { CognitoJwtVerifier } = require("aws-jwt-verify");
+const AWS = require("aws-sdk");
+
+// Verifier that expects valid access tokens:
+const verifier = CognitoJwtVerifier.create({
+  userPoolId: process.env.poolId,
+  tokenUse: "access",
+  clientId: process.env.clientId,
+});
+
 module.exports.handler = async (event) => {
-  const AWS = require('aws-sdk')
-  const s3 = new AWS.S3({apiVersion: '2006-03-01'});
+  try {
+    await verifier.verify(event.headers["Authorization"]);
```

Comment:
Shouldn't we need the secret key to verify the token?

Reply:
They are in the pool that it connects to by the instance of the verifier above